### ISSUE TITLE
quality about the par_sav

### ISSUE THREAD

**Issue by Jackson-1102** [Zhangjinso/PISE#33]
I am curiosity about the quality of the output parsing map of p2.And I Try to train for several days on V100.
The final png is so good.But the par_sav is different from the SPL2.
![image](https://user-images.githubusercontent.com/66883561/129370052-54941165-4e7e-471e-b119-76dcee9d2792.png)

![image](https://user-images.githubusercontent.com/66883561/129370107-ad4e9c48-8eb4-4ffb-b862-453121ef1e53.png)

![image](https://user-images.githubusercontent.com/66883561/129370197-2e9af7a6-15e3-4c3c-acef-48fbc4bf78a7.png)

![image](https://user-images.githubusercontent.com/66883561/129370235-f8288e2b-727f-4c2b-ac0d-8cee8f2da20d.png)

And I have seen the constrains over them,which should helps to make them very similiar.But how this happends with no head,no skin in the img.
And I want to ask for whether the coordconv is  useful.

**Comment by Zhangjinso** [comment]
Hi,
for the first question, I guess it may be due to using the logits as the 'parsing result' for the image generator for human pose transfer, which may lead to a lower loss with limited regions (it seems due to the upper clothes and lower clothes are the larger region of the human parsing). 
For the second question, we use coordconv in spatial-aware normalization, which is useful, like position encoding, to learn spatial correspondence in our opinion. But we did not conduct an ablation study about this part.
